Marlin/ultralcd.cpp

```diff
@@ -862,7 +862,8 @@ static void lcd_move_e(
   if (encoderPosition != 0) {
     current_position[E_AXIS] += float((int)encoderPosition) * move_menu_scale;
     encoderPosition = 0;
-    line_to_current(E_AXIS);
+    if (movesplanned() <= 3)
+      line_to_current(E_AXIS);
     lcdDrawUpdate = 1;
   }
```

Comment:
As this is, it will drop moves, and those moves will stay dropped when the LCD times out and returns to the info screen. However, it turns out there is an easy solution here to avoid dropping any moves:
```cpp
    if (encoderPosition != 0 && movesplanned() <= 3) {
      current_position[E_AXIS] += float((int)encoderPosition) * move_menu_scale;
      line_to_current(E_AXIS);
      encoderPosition = 0;
      lcdDrawUpdate = 1;
    }
```
This holds onto the encoder position and doesn't reset it, so the next time this is invoked, the move will still be pending. With this change you can throw away the `line_to_current` call with `LCD_CLICKED`.